packages/client/src/builders/search/facetBuilder.ts

```diff
@@ -357,6 +358,29 @@ export class FacetBuilder {
 
         return this;
     }
+
+    public addRecentlyPurchasedFacet(
+        purchaseQualifiers: PurchaseQualifiers,
+        selectedValues: boolean[] | null = null,
+        facetSettings?: FacetSettings): this {
+
+        const facet: RecentlyPurchasedFacet = {
+            $type: 'Relewise.Client.DataTypes.Search.Facets.Queries.RecentlyPurchasedFacet, Relewise.Client',
+            field: 'Data',
+            settings: facetSettings,
+            selected: selectedValues,
+            purchaseQualifiers: {
+                byUser: purchaseQualifiers.byUser,
+                byUserCompany: purchaseQualifiers.byUserCompany,
+                byUserParentCompany: purchaseQualifiers.byUserParentCompany,
+                sinceMinutesAgo: purchaseQualifiers.sinceMinutesAgo,
+            },
```

Comment:
Can we just do this?

```suggestion
            purchaseQualifiers: purchaseQualifiers,
```